src/main/java/org/spongepowered/api/event/world/chunk/WorldChunkEvent.java

```diff
@@ -28,54 +28,54 @@
 import org.spongepowered.api.event.Cancellable;
 import org.spongepowered.api.event.Event;
 import org.spongepowered.api.world.World;
-import org.spongepowered.api.world.chunk.Chunk;
+import org.spongepowered.api.world.chunk.WorldChunk;
 import org.spongepowered.math.vector.Vector3i;
 
-public interface ChunkEvent extends Event {
+public interface WorldChunkEvent extends Event {
```

Comment:
@Zidane I've opted to make this rename here because all event really are on `WorldChunk`s, but we might for simplicity not want this change. If not - easy to revert.

Reply:
I'd leave it as ChunkEvent and refactor it so that the base event doesn't supply the World's key.